tests/framework/Optimizers/GradientDescent/basic.xml

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="debug">
+  <TestInfo>
+    <name>framework/Optimizers.Basic</name>
+    <author>talbpaul</author>
+    <created>2020-01-29</created>
+    <classesTested>Optimizer</classesTested>
+    <description>
+      This is the most basic test of the Optimizer. It employs finite difference for the gradient
+      estimation, gradient history for the step determination, and strict improvement acceptance. It
+      converges only on the number of samples taken with a single trajectory on the Beale function. It
+      also demonstrates typical outstream options for optimization. It also tests passing through
+      constants and functions.
+    </description>
+    <analytic>
+      This test uses Beale's function, which is documented in the analytic tests documentation under
+      the Optimizer functions section.
+    </analytic>
```

Comment:
requirement test?

Reply:
it should be

Reply:
I intend each of the tests to test a part of the optimizer; should they all be requirements tests?

Reply:
I need to know what list of features is requirements based, then I can go find and update the docs and tests. Do we have this level of requirements tests throughout the code base? This will mean the Chair or designee is the only reviewer for many changes to the optimizer.

Reply:
Nono Just the Basic and the bound contraints one... we should have a requirement (R-RM-1) about optimization and I think we "tag" some of the old opt tests with it.

Reply:
The "minimal" test is tagged with RM-1 right now, I will add the boundary and functional constraints; did you want to add another req that "basic" covers?

Reply:
perfect for now...